resources/config/distribution.php

```diff
@@ -11,7 +11,7 @@
     |
     */
 
-    'name'        => 'PHP Classifieds Marketplace software',
-    'description' => 'OpenClassify is modular and most advanced open source classified platform build with Laravel.Use it in real estate classifieds, auto, e-commerce and other ...',
+    'name'        => '',
+    'description' => '',
```

Comment:
![image](https://user-images.githubusercontent.com/39536659/149896440-d2a3d21b-43c6-420b-8c2c-efd9372216d7.png)


This field comes when the system name field is empty.You can assign a default system name instead.Or you can change the areas where it is used. @diashalabi @profstyle1